packages/python-chardet/python-chardet.spec

```diff
@@ -5,8 +5,8 @@
 %global pypi_name chardet
 
 Name:           %{?scl_prefix}python-%{pypi_name}
-Version:        3.0.4
-Release:        3%{?dist}
+Version:        4.0.0
```

Comment:
aiohttp-3.7.4 requires chardet < 4.0, so this will break it.

post0 *does* allow 4.0, but your commit above didn't update the boundaries for some reason.

Reply:
I'll drop this as well